scripts/hum_herb.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import numpy
+import logging
+import herbpy
+import humanpy
+
+if __name__ == "__main__":
+    
+    logger = logging.getLogger('test_humherb')
+    logger.setLevel(logging.INFO)
+    
+    sim = True
+    attach_viewer = True
+    env, herb = herbpy.initialize(attach_viewer=attach_viewer, sim=sim)  
+
+    _, human = humanpy.initialize(attach_viewer=attach_viewer, sim=sim, env=env)    
```

Comment:
I advise against passing `attach_viewer` to both `initialize` functions because it could result in `SetViewer` being called twice on `env`. I am not sure what effect this will have - `openravepy` is quite picky about having *one, and only one* viewer.